DEV_SETUP.md

```diff
@@ -167,28 +171,54 @@ Create the shared directory.  If you have not modified `SHARED_DRIVE_ROOT`, then
 Once you have completed the above steps, you can use Docker to build
 and run all of the service containers. There are detailed instructions
 for setting up Docker in the [docker folder](docker/README.md). But the
-following should cover the needs of most developers:
+following should cover the needs of most developers.
+
 
+1. Install docker packages.
+
+    ```sh
     $ sudo apt install docker.io
     $ pip install docker-compose
     $ sudo adduser $USER docker
+    ```
 
-Log in as yourself again, to activate membership of the "docker" group:
+1. Log in as yourself again to activate membership of the "docker" group and
+   re-activate your virtualenv.
 
+    ```sh
     $ su - $USER
+    $ workon cchq
+    ```
+
+1. Ensure the Docker service is running.
 
-Ensure the Docker service is running:
+    ```sh
+    $ sudo systemctl status docker
+    # if service is not running, execute:
+    $ sudo systemctl start docker
+    ```
 
-    $ sudo service docker status
+1. Ensure the elasticsearch config files are world-readable (their containers
+   will fail to start otherwise).
 
-Bring up the Docker containers for the services you probably need:
+    ```sh
+    chmod 0644 ./docker/files/elasticsearch*.yml
+    ```
 
-    $ scripts/docker up postgres couch redis elasticsearch zookeeper kafka minio
```

Comment:
I think it is still valuable to know which services you should have running once `scripts/docker up` is run

Reply:
This list is all of the containers except for `formplayer`. I thought it might be clearer to just "bring them all up" and then add a note about why you might not want some of them running (see added step about formplayer below).  Perhaps it would be more informative to provide a command that lists all the containers that will be started and add docs about why each one is needed?

Reply:
Ah I missed that. Looks good.